docs/containers-registries.conf.5.md

```diff
@@ -244,6 +250,8 @@ Given the above, a pull of `example.com/foo/image:latest` will try:
 
 in order, and use the first one that exists.
 
+Note that a mirror is associated only with the previous registry.  Pulling `registry.com/image:latest` will hence only reach out to `mirror.registry.com` and not use the one associated to the other registry.
```

Comment:
Again, consider something like “`[[registry]]` TOML table” rather than just “registry” (in both sentences). Also `s/previous/current/` — `example.com/foo` is the “previous” one.

Or maybe say something to express the idea that “a [[registry.mirror]] TOML table is, per TOML semantics, logically nested inside the previous [[registry]] table, and only applies to that [[registry]] table” — to talk about the options and their nesting, instead of relative ordering of tables as if they were conceptually unrelated. I don’t have a good idea of which one is easier to understand.